MetaMorpheus/EngineLayer/Gptmd/GptmdEngine.cs

```diff
@@ -60,22 +62,21 @@ public static bool ModFits(Modification attemptToLocalize, IBioPolymer protein,
             return false;
         }
 
+
         protected override MetaMorpheusEngineResults RunSpecific()
         {
             var modDict = new ConcurrentDictionary<string, ConcurrentBag<Tuple<int, Modification>>>();
             int modsAdded = 0;
```

Comment:
You should make the modDict a property instead of a local variable. This prevents it from being duplicated for each thread.

Reply:
i dont' know what you mean. can you explain or show me?